marketplace-service/src/main/java/ch/akros/marketplace/service/service/TopicService.java

```diff
@@ -337,6 +369,45 @@ private List<TopicSearchResponseDTO> filterTopicsByToPrice(String topicValueToPr
         return topicList;
     }
 
+    private List<TopicSearchResponseDTO> filterTopicsByFromDate(String topicValueFromDate, 
+                   List<TopicSearchResponseDTO> topicList) {
+        
+        String[] fromDate = topicValueFromDate.split("-", 3);
```

Comment:
May be it is better to use here the date API - LocalDate. 
Something like this:
LocalDate date = LocalDate.parse(topicValueFromDate");

Reply:
You're right. the code was rewritten. Thanks for the advice.